packages/react/src/components/SpinButton/SpinButton.base.tsx

```diff
@@ -447,9 +447,8 @@ export const SpinButtonBase: React.FunctionComponent<ISpinButtonProps> = React.f
           autoComplete="off"
           role="spinbutton"
           aria-labelledby={label && labelId}
-          // TODO: test what happens while editing
-          aria-valuenow={ariaValueNow ?? (valueIsNumber ? Number(value) : undefined)}
           aria-valuetext={ariaValueText ?? (valueIsNumber ? undefined : value)}
+          aria-valuenow={ariaValueNow}
```

Comment:
Should this be removed?

Reply:
`ariaValueNow` is a prop the author would need to explicitly pass in. I don't think we should've made that part of the API in the first place, but I don't think it's worth deprecating now.

Hopefully not many people are using it, and at least now `aria-valuenow` won't be defined by default :P